src/styles/GreetingScreen/Screen_Greeting.jsx

```diff
@@ -0,0 +1,45 @@
+import { StyleSheet } from 'react-native';
+
+export const stylesGreeting = StyleSheet.create({
+  containerLogo: {
+    alignItems: 'center',
+    alignSelf: 'center',
+    marginTop: 90
+  },
+
+  containerTexts: {
+    marginTop: 130
+  },
+
+  text1: {
+    color: 'white', 
+    fontFamily: 'Nunito_500Medium',
+    fontSize: 30,
+    fontWeight: 'bold'
+  },
+
+  text2: {
+    color: 'white', 
+    fontFamily: 'Nunito_500Medium',
+    fontSize: 30, 
+    fontWeight: 'bold',
+    marginLeft:35
+  },
```

Comment:
You have here duplicated code, lets use the text1 styles in your tex2 and just add the marginLeft style to `text2`